app/code/Magento/NewRelicReporting/Test/Unit/Model/NewRelicWrapperTest.php

```diff
@@ -0,0 +1,197 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+declare(strict_types=1);
+
+namespace Magento\NewRelicReporting\Test\Unit\Model;
+
+use Magento\Framework\TestFramework\Unit\Helper\ObjectManager;
+use Magento\NewRelicReporting\Model\NewRelicWrapper;
+use PHPUnit\Framework\TestCase;
+
+/**
+ * Override system method for assertions
+ *
+ * @param string $extensionName
+ * @return bool
+ */
+function extension_loaded(string $extensionName): bool
+{
+    NewRelicWrapperTest::$systemMethodsCalled['extension_loaded'] = [$extensionName];
+
+    return NewRelicWrapperTest::$stubIsNewrelicEnabled;
+}
+
+/**
+ * Override system method for assertions
+ *
+ * @param string $transactionName
+ */
+function newrelic_name_transaction(string $transactionName): void
+{
+    NewRelicWrapperTest::$systemMethodsCalled['newrelic_name_transaction'] = [$transactionName];
+}
+
+/**
+ * Override system method for assertions
+ *
+ * @param bool $ignore
+ */
+function newrelic_end_transaction(bool $ignore = false): void
+{
+    NewRelicWrapperTest::$systemMethodsCalled['newrelic_end_transaction'] = [$ignore];
+}
+
+/**
+ * Override system method for assertions
+ *
+ * @param string $appName
+ */
+function newrelic_set_appname(string $appName): void
+{
+    NewRelicWrapperTest::$systemMethodsCalled['newrelic_set_appname'] = [$appName];
+}
+
+class NewRelicWrapperTest extends TestCase
+{
+    const STUB_TRANSACTION_NAME = 'TransactionName';
+    const STUB_APPLICATION_NAME = 'ApplicationName';
+
+    /**
+     * @var bool
+     */
+    public static $stubIsNewrelicEnabled;
+
+    /**
+     * @var array[]
+     */
+    public static $systemMethodsCalled;
+
+    /**
+     * @var NewRelicWrapper
+     */
+    private $wrapper;
+
+    /**
+     * Flush System state to entry
+     */
+    public function setUp()
+    {
+        $objectManager = new ObjectManager($this);
+        $this->wrapper = $objectManager->getObject(NewRelicWrapper::class);
+
+        self::$systemMethodsCalled = [];
+    }
+
+    /**
+     * When NewRelic extension is not loaded, `newrelic_set_appname` should not be called
+     */
+    public function testSetAppNameWhenExtensionNotLoaded()
+    {
+        $this->markTestSkipped('Magento Coding Standard does not allow to mock System methods');
```

Comment:
Please remove skipping tests, if static tests are complains about this file - let's ignore specific issues, but we don't need to prevent unit tests to be executed

Reply:
Unit Tests were removed as do not introduce any value.